src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2157,7 +2157,6 @@ public override BoundNode VisitForEachStatement(BoundForEachStatement node)
             {
                 _pendingBranches.Add(new PendingBranch(node, this.State));
             }
-            //if (_trackExceptions) NotePossibleException(node); // PROTOTYPE(async-streams)
```

Comment:
:memo: I've created a PR against `master` to remove `_trackExceptions` entirely.